sdk/.gitignore

```diff
@@ -0,0 +1,2 @@
+# used for azure tts generation in a detached branch.
+scripts
```

Comment:
What's this script folder for?

Reply:
Should be not needed, I used it for a tts generate audio file storage, will remove this change, thanks.

Reply:
@olivershen-wow removed the file, please help review again thanks.